<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRD2</class>
 <widget class="QWidget" name="XRD2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>319</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image and region</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QComboBox" name="image">
        <property name="neutrinoImage" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="rectROI">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../build-neutrinoMaster-Brew-Debug/src/qrc/icons.qrc">
          <normaloff>:/icons/rect.png</normaloff>:/icons/rect.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Transformations</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Rotate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Flip LeftRight</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="flipLR">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../build-neutrinoMaster-Brew-Debug/src/qrc/icons.qrc">
          <normaloff>:/icons/flipLeftRight.png</normaloff>:/icons/flipLeftRight.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="neutrinoSave" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Flip UpDown</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QToolButton" name="flipUD">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../build-neutrinoMaster-Brew-Debug/src/qrc/icons.qrc">
          <normaloff>:/icons/flipUpDown.png</normaloff>:/icons/flipUpDown.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="neutrinoSave" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Transpose</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QToolButton" name="transpose">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../../build-neutrinoMaster-Brew-Debug/src/qrc/icons.qrc">
          <normaloff>:/icons/transpose.png</normaloff>:/icons/transpose.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="neutrinoSave" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="angle">
        <property name="toolTip">
         <string>Fringes orientation</string>
        </property>
        <property name="suffix">
         <string>Ëš</string>
        </property>
        <property name="minimum">
         <double>-360.000000000000000</double>
        </property>
        <property name="maximum">
         <double>360.000000000000000</double>
        </property>
        <property name="neutrinoSave" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="name">
        <property name="neutrinoSave" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="original">
        <property name="text">
         <string>Show Original</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="show">
        <property name="text">
         <string>Show Transformed</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>image</tabstop>
  <tabstop>rectROI</tabstop>
  <tabstop>angle</tabstop>
  <tabstop>flipLR</tabstop>
  <tabstop>flipUD</tabstop>
  <tabstop>transpose</tabstop>
  <tabstop>name</tabstop>
  <tabstop>original</tabstop>
  <tabstop>show</tabstop>
 </tabstops>
 <resources>
  <include location="../../../build-neutrinoMaster-Brew-Debug/src/qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
